tests/integration/test_logtest/test_log_process_options/test_rules_verbose.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import shutil
+import re
+
+import pytest
+import yaml
+from wazuh_testing.tools import WAZUH_PATH, LOGTEST_SOCKET_PATH, WAZUH_UNIX_USER, WAZUH_UNIX_GROUP, LOCAL_RULES_PATH
+
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+# Configurations
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data/')
+messages_path = os.path.join(test_data_path, 'rules_verbose.yaml')
+with open(messages_path) as f:
+    test_cases = yaml.safe_load(f)
+
+# Variables
+receiver_sockets_params = [(LOGTEST_SOCKET_PATH, 'AF_UNIX', 'TCP')]
+
+local_rules_debug_messages = ['Trying rule: 880000 - Parent rules verbose', '*Rule 880000 matched',
+                              '*Trying child rules', 'Trying rule: 880001 - test last_match', '*Rule 880001 matched',
+                              '*Trying child rules', 'Trying rule: 880002 - test_child test_child']
+
+
+# Fixtures
+@pytest.fixture(scope='function')
+def configure_rules_list(get_configuration, request):
+    """
+    Configure a custom rules for testing.
+    Restart Wazuh is not needed for applying the configuration is optional.
+    """
+
+    # save current rules
+    shutil.copy(LOCAL_RULES_PATH, LOCAL_RULES_PATH+'.cpy')
+
+    file_test = get_configuration['rule_file']
+    # copy test rules
+    shutil.copy(test_data_path + file_test, LOCAL_RULES_PATH)
+    shutil.chown(LOCAL_RULES_PATH, WAZUH_UNIX_USER, WAZUH_UNIX_GROUP)
+
+    yield
+
+    # restore previous configuration
+    shutil.move(LOCAL_RULES_PATH+'.cpy', LOCAL_RULES_PATH)
+    shutil.chown(LOCAL_RULES_PATH, WAZUH_UNIX_USER, WAZUH_UNIX_GROUP)
+
+
+@pytest.fixture(scope='module', params=test_cases, ids=[test_case['name'] for test_case in test_cases])
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Tests
+def test_rules_verbose(get_configuration, configure_rules_list, connect_to_sockets_function):
```

Comment:
This test did not use any fixture to up necessary wazuh modules. This makes this test depends on the rest of the suite.

Please create or use an existing fixture to up minimum modules to use for this test module.

Reply:
In this case, this test only needs to up required daemons. So we can create the following fixture:
```
@pytest.fixture(scope='module')
def restart_required_logtest_daemons(request):
    """Wazuh daemons handler.
    """
    required_logtest_daemons = ['wazuh-analysisd', 'wazuh-monitord']


    for daemon in required_logtest_daemons:
        control_service('restart', daemon=daemon)

    for i in required_logtest_daemons:
        check_daemon_status(daemon)

    yield 

    for i in required_logtest_daemons:
        control_service('stop', daemon=daemon)

```

And calls it in the test

```
# Tests
def test_rules_verbose(restart_required_logtest_daemons, get_configuration, configure_rules_list, 
                       connect_to_sockets_function):
```